debugtools/DDR_VM/src/com/ibm/j9ddr/vm29/tools/ddrinteractive/commands/J9BCUtil.java

```diff
@@ -728,6 +731,21 @@ private static void dumpClassJ9ExtraModifiers(PrintStream out, long accessFlags)
 			out.append("(sealed) ");
 	}
 
+	private static void dumpMethodJ9Modifiers(PrintStream out, long modifiers, long extraModifiers) {
```

Comment:
Rather than `long`, perhaps `UDATA` is a better type for `modifiers` and `extraModifiers`, then this method can use `Scalar.allBitsIn(long)` which properly handles the case where the argument is zero, for example:
```
		if (modifiers.allBitsIn(J9JavaAccessFlags.J9AccMethodCallerSensitive)) {
			out.append(" (@CallerSensitive)");
		}
```